CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ release.
   ([#3402](https://github.com/open-telemetry/opentelemetry-specification/pull/3402))
     BREAKING: rename `net.peer.name` to `server.address` on client side and to `client.address` on server side,
      `net.peer.port` to `server.port` on client side and to `client.port` on server side,
-     `net.host.name` and `net.host.port` to `server.name` and `server.port` (since `net.host.*` attributes only applied to server instrumentation)
+     `net.host.name` and `net.host.port` to `server.address` and `server.port` (since `net.host.*` attributes only applied on server side)
```

Comment:
I just changed this for consistency, but it's actually wrong.

All the net.host.* attributes apply independent of whether something is a client, a server, or neither (P2P, etc.). They are not defined with dependencies on these concepts.